docs/docs/gatsby-style-guide.md

```diff
@@ -278,3 +287,151 @@ The Hemingway App will assign a “grade level” for your writing. You should a
+
+Attributes:
+
+- new to React
+- interested in React
+- new to Gatsby
+- interested in Gatsby
+- proficient with browsers and operating systems
+
+Looking for:
+
+- a way to learn and/or improve React skills
+- a way to start a site and/or app project that uses React
+
+### Tutorial purpose
+
+By following the steps in the tutorial, a user should:
+
+- Experience the value of Gatsby as quickly as possible. With Gatsby, a user typically values that it takes fewer steps (and is therefore easier) to:
+  - start coding immediately without being an expert
+  - start a new project
+  - make edits and see them thru hot reloading
+  - publish a site
+  - do basic tasks like create pages, link between pages, create routing, change styles
+- Know how to and actually start and deploy a site as quickly as possible
+- Be able to share their site
+- Know how to and actually find more advanced tutorials and docs
+- Have fun!
+- Use enough React to do basic tasks like creating pages, links, styles
+
+### Tutorial tone and style
+
+The tone and style of the tutorial should effectively help the audience reach their goals.
+
+### Use personal "you" and be warm
+
+The main tutorial ought to use the same personal “you” like the rest of the docs; in addition, the tutorial ought to use a warm, validating tone by congratulating users, complimenting them, and generally saying things like “yay!” more often.
+
+> Why not use "yay" in the docs as well? Since the tutorial's goal is to help users complete a series of steps, it is possible and helpful to congratulate them on successfully completing each step. Guides in the docs act as reference guides that users can browse at will rather than read from top to bottom. It doesn't make sense to congratulate someone on finishing a guide, since it's not a series of steps.
+
+#### Don't make users think more than is necessary
+
+Because the audience of the tutorial is people who do not consider themselves experts in React, it's important to reduce the amount of new information to bare minimum. The goal: give people only the information necessary to complete a task and to know how to repeat the task again, outside of the context of the tutorial.
+
+In practice, you can reach this goal by two rules of thumb:
+
+- reduce the number of hyperlinks, tabs, and environments to the least number required to complete the tasks in the tutorial.
+- when there are multiple ways to complete a task, give people only one way. This way ought to be the best practice possible within the constraints of the lowest supported versions of software. If the best practice isn't possible with the lowest supported versions of software, mention that as a side note.
+
+## Docs
+
+While the main tutorial is optimized for users who are not experts in React and/or Javascript, the `/docs/` are optimized for those with intermediate to expert mastery with React and Javascript. The purpose, tone, and style of the docs ought to align with this audience.
```

Comment:
Yeah, just because someone could skip just to this section. I think it's not necessary though, b/c if someone wants to know the difference, they'll just go to the header you mentioned. Good catch!